plugins/auto-sizes/tests/test-auto-sizes.php

```diff
@@ -32,12 +32,6 @@ public function get_image_tag( int $attachment_id ): string {
 		return get_image_tag( $attachment_id, '', '', '', 'large' );
 	}
 
-	public function test_hooks(): void {
-		$this->assertSame( 10, has_filter( 'wp_get_attachment_image_attributes', 'auto_sizes_update_image_attributes' ) );
-		$this->assertSame( 10, has_filter( 'wp_content_img_tag', 'auto_sizes_update_content_img_tag' ) );
-		$this->assertSame( 10, has_action( 'wp_head', 'auto_sizes_render_generator' ) );
-	}
```

Comment:
Nit-pick, but technically it would be more appropriate to keep this test but make it behave differently based on whether the Core function exists. If it does not exist, the same assertions apply like now. If it does exist, you could assert that the first two filters are _not_ added.

Note also that the generator filter is relevant either way, so that's another reason not to remove this test.

Reply:
As I suggested in https://github.com/WordPress/performance/pull/1547#discussion_r1768975834 correct?